buildSrc/src/main/kotlin/buildsrc/convention/version-management.gradle.kts

```diff
@@ -0,0 +1,41 @@
+package buildsrc.convention
+
+import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
+
+plugins {
+    base
+    id("com.github.ben-manes.versions")
+    id("se.patrikerdes.use-latest-versions")
+}
```

Comment:
I'm not sure how these plugins are supposed to work, so I'm not sure if they're set up correctly. 

I think it might be worth migrating to use `libs.version.toml` and  https://github.com/littlerobots/version-catalog-update-plugin

Reply:
Absolutely, to have the Deps in the build source folder is a relict of gradle didn't having a version catalog feature

Reply:
okay great. I'll leave it be for the moment.